.github/workflows/testing.yml

```diff
@@ -899,4 +899,42 @@ jobs:
         run: ./gen_grpc.sh
 
       - name: Run Tests
-        run: make jupyter
\ No newline at end of file
+        run: make jupyter
+
+  schema-changer:
+    name: Schema Changer Testing
+    needs: setup
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download Working Compiled Directories
+        uses: actions/download-artifact@v3
+        with:
+          name: compiled-workdir
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+          
+      - name: Install ETCD
+        run: |
+          git clone -b v3.4.16 https://github.com/etcd-io/etcd.git
+          cd etcd
+          ./build
+          export PATH="$PATH:`pwd`/bin"
+          etcd --version
+          etcd --logger=zap &
+
+
+      - name: Insert Values
+        working-directory: ./helpers/schema_changer
+        run: go run load.go
+
+      - name: Convert Values
+        working-directory: ./helpers/schema_changer
+        run: go run convert.go
```

Comment:
where is convert.go?, will that be added in implementation?

Reply:
Yeah this PR is delayed for now though. Will close